target/linux/sifiveu/image/Config.in

```diff
@@ -0,0 +1,5 @@
+config SIFIVEU_SD_BOOT_PARTSIZE
```

Comment:
In the future we should make `TARGET_SD_BOOT_PARTSIZE` a common option shared by multiple target.

Reply:
I booted the system build by OpenWrt using the vendor bootloader over the network. 
The SoC has some strange/ different DMA behavior which is not supported upstream. I am not sure if this breaks the SiFive SoCs.
I tried to only backport some good looking patches, but then it booted only sometimes.

Reply:
Currently I have all vendor patches, We can definitely remove some of them, but we need some dirty kernel patches.

I haven't looked at the U-Boot and SBI.

Reply:
@hauke @wigyori  Hi all，thanks for sharing your source code.  I'm very interested in openwrt. and I have installed the openwrt on a x86 device. So amazing! I use so many useful packages,like Wake on Lan、 ddns-ipv6、Frp server、vpn、ADguard, etc. I have a VisionFive board too and I want to port the openwrt to the  board, but I dont have any experience，I watched hauke's  presentation  on Youtube （Adding new SoC to OpenWrt by Hauke Mehrtens - OpenWrt Summit）、  I downloaded wigyori‘s codes. I want to use the  vendor linux kernel trees as the externel kernel tree to compile, but  It doesn't work......

I‘ve ported openwrt to VisionFive board for about 4 weeks, but I don’t make any progress，so frustrated for me. 
≡(▔﹏▔)≡

I'm so interested in openwrt and I’m eager to learn from you! Can you give me some suggestions？
Please do not hesitate to contact me if I can help. 
caoyaqing163@qq.com
Thanks！

Reply:
Does there have any guide for visionfive build a sdcard openwrt image?